From 2d2d46dafd5f55796fc7a17f38dd1ee2be051ebc Mon Sep 17 00:00:00 2001
From: Alessandro Piras <laynor@gmail.com>
Date: Thu, 19 Jan 2012 21:18:49 +0100
Subject: [PATCH] Took off other client-messages stuff

---
 trayion/systray.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/trayion/systray.c b/trayion/systray.c
index 360da3c..c4f03a2 100644
--- a/trayion/systray.c
+++ b/trayion/systray.c
@@ -29,11 +29,9 @@
 #include "sorted_classes.h"
 #include "hidden_list.h"
 #include "xutils.h"
-#include "client_messages.h"
 
 static Atom systray_atom = None;
 static Atom opcode_atom = None;
-Atom client_event_atom = None;
 static Atom message_atom = None;
 struct list_head systray_list;
 struct list_head *current_item = &systray_list;
@@ -69,7 +67,6 @@ int init_systray() {
 	opcode_atom = XInternAtom (main_disp, "_NET_SYSTEM_TRAY_OPCODE", False);
 	message_atom = XInternAtom (main_disp, "_NET_SYSTEM_TRAY_MESSAGE_DATA",
 				    False);
-	client_event_atom = XInternAtom(main_disp, CLIENT_EVENT_ATOM_NAME, False);
 
 	/*
 	 * Selection managers are required to broadcast their existence when
