diff --git a/Makefile b/Makefile
index c95aa24..7b5fb23 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,7 @@
-prefix=/usr
-bindir = ${prefix}/bin
-mandir = ${prefix}/share/man
+DESTDIR ?= "/"
+prefix ?= "usr"
+bindir ?= "${prefix}/bin"
+mandir ?= "${prefix}/share/man"
 INSTALL = /usr/bin/install
 INSTALLPROG = ${INSTALL} -m 0755
 INSTALLMAN = ${INSTALL} -m 0644
@@ -14,9 +15,9 @@ clean:
 	make -C trayion clean
 
 install: trayion/trayion
-	mkdir -p ${bindir}
-	mkdir -p ${mandir}/man1
-	${INSTALLPROG} trayion/trayion ${bindir}/trayion
-	${INSTALLPROG} trayion-dump.sh ${bindir}/trayion-dump
-	${INSTALLPROG} trayion-reload.sh ${bindir}/trayion-reload
-	${INSTALLMAN} doc/trayion.1 ${mandir}/man1/trayion.1
+	mkdir -p ${DESTDIR}${bindir}
+	mkdir -p ${DESTDIR}${mandir}/man1
+	${INSTALLPROG} trayion/trayion ${DESTDIR}${bindir}/trayion
+	${INSTALLPROG} trayion-dump.sh ${DESTDIR}${bindir}/trayion-dump
+	${INSTALLPROG} trayion-reload.sh ${DESTDIR}${bindir}/trayion-reload
+	${INSTALLMAN} doc/trayion.1 ${DESTDIR}${mandir}/man1/trayion.1
